### ISSUE TITLE
Camera drift issue still a problem

### ISSUE THREAD

**Issue by TheRealSyalin** [armory3d/armortools#1553]
https://github.com/armory3d/armortools/issues/450

A post has already bin made about this, but it was a while ago and i cant seem to find a fix. The bug still happens in the new dev build and in 08

**Comment by fazlee712** [comment]
yeah can confirm, I've just conditioned myself to press Num0 in between every operation involving the mouse to get around it for the last few months haha

**Comment by Corruptinator** [comment]
Can confirm as well, movement happens through right click

**Comment by 4beraser** [comment]
same issues here, but reset camera do fix the issue, so not a big problem I guess?

**Comment by luboslenco** [comment]
It should be resolved in the git version, but will still keep an eye on it. Will update pre-built binaries shortly. Thanks for reporting this. Linking with https://github.com/armory3d/armortools/issues/1823.
